### PR TITLE
Fugu improvements

### PR DESCRIPTION
**Pull request by ghost**
Verb > Action button

Sprite no longer transforms, stops the sprite from becoming blurry

t. wumbology

### CODE REVIEW

**Comment by Armhulen** [comment]
I wasn't that much against this PR, but OP appears to be wumbo. That earns a solid :-1: from me.

**Comment by Supermichael777** [comment]
wow another opportunity to bully difbot and its inability to grasp change.

**Comment by Armhulen** [comment]
I wasn't that much against IconDiffBot, but it appears to be an retarded. That earns a solid :thumbsdown: from me

**Comment by Buggy123** [comment]
I wasn't that much against Armhulen, but it appears to be an retarded. That earns a solid :-1: from me

**Comment by Armhulen** [comment]
okay fair enough i'll limit it to one per PR

**Comment by Repukan** [comment]
Fugu more like

Fug u

**Comment by Armhulen** [comment]
classic repukan

**Comment by Repukan** [comment]
Bada bing

File: code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm [hunk 1]
```diff
@@ -46,42 +57,47 @@
...
+/datum/action/innate/fugu/expand/Activate()
+	var/mob/living/simple_animal/hostile/asteroid/fugu/F = owner
+	if(F.wumbo)
+		to_chat(F, "<span class='notice'>YOU'RE ALREADY WUMBO!</span>")
 		return
-	if(inflate_cooldown)
-		to_chat(src, "<span class='notice'>We need time to gather our strength.</span>")
+	if(F.inflate_cooldown)
+		to_chat(F, "<span class='notice'>You need time to gather our strength.</span>")
 		return
```
Comment (nicbn) [inline code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm]: You need time to gather **your** strength

Comment (Armhulen) [reply code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm]: ty

**Comment by Armhulen** [comment]
I need a grammar tag, code improvement tag and maybe fix if you're generous it was really hard to make the fugu sprite bigger for some reason idk

**Comment by Ergovisavi** [comment]
>Someone actually remembers this ancient-ass mob I put in the fucking codebase

What the fuck

**Comment by Bawhoppen** [comment]
Lots of people know what it is?

**Comment by Doppeltongue** [comment]
It shows up in xenobiology as one of the funnier mobs from gold slimes.

**Comment by Armhulen** [comment]
It's in xenobio

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**